### ISSUE TITLE
How to finetune the NEST model with CTC Loss for ASR task?

### ISSUE THREAD

**Issue by zw76859420** [NVIDIA/NeMo#11163]
We have trained SSL model based on the NEST, how to finetune the previous model based on the CTC Loss function?
The pretraining scripts  are as follows: [NEST](https://github.com/NVIDIA/NeMo/blob/main/examples/asr/speech_pretraining/masked_token_pred_pretrain.py)
